packages/docusaurus-plugin-content-blog/src/__tests__/index.test.ts

```diff
@@ -541,4 +541,19 @@ describe('blog plugin', () => {
 
     expect(blogTags).toMatchSnapshot();
   });
+
+  it('process blog posts load content', async () => {
+    const siteDir = path.join(
+      __dirname,
+      '__fixtures__',
+      'website-blog-with-dates',
+    );
```

Comment:
I think for this one you can just reuse an existing site fixture, just do something like `limit(blogPosts,4)` and assert that it impacts things such as pagination + take a snapshot